repondeur/tests/conftest.py

```diff
@@ -44,100 +44,1169 @@ def mock_dossiers():
+        "libelleEdition": "de la commission des finances",
+        "libelleAbrege": "Finances",
+        "libelleAbrev": "CION_FIN",
+        "viMoDe": {"dateDebut": "1958-12-09", "dateAgrement": None, "dateFin": None},
+        "organeParent": None,
+        "chambre": None,
+        "regime": "5ème République",
+        "legislature": None,
+        "secretariat": {
+            "secretaire01": "Mme Dominique Meunier-Ferry",
+            "secretaire02": None,
+        },
+    },
+    "PO717460": {
+        "@xsi:type": "OrganeParlementaire_Type",
+        "uid": "PO717460",
+        "codeType": "ASSEMBLEE",
+        "libelle": "Assemblée nationale de la 15ème législature",
+        "libelleEdition": "de l'Assemblée",
+        "libelleAbrege": "Assemblée",
+        "libelleAbrev": "AN",
+        "viMoDe": {"dateDebut": "2017-06-21", "dateAgrement": None, "dateFin": None},
+        "organeParent": None,
+        "chambre": None,
+        "regime": "5ème République",
+        "legislature": "15",
+        "secretariat": {"secretaire01": None, "secretaire02": None},
+    },
+    "PO78718": {
+        "@xsi:type": "OrganeExterne_Type",
+        "uid": "PO78718",
+        "codeType": "SENAT",
+        "libelle": "Sénat ( 5ème République )",
+        "libelleEdition": "du sénat ( 5ème République )",
+        "libelleAbrege": "Sénat",
+        "libelleAbrev": "SENAT",
+        "viMoDe": {"dateDebut": "1958-12-09", "dateAgrement": None, "dateFin": None},
+        "organeParent": None,
+    },
+    "PO730934": {
+        "@xsi:type": "GroupePolitique_type",
+        "uid": "PO730934",
+        "codeType": "GP",
+        "libelle": "Les Républicains",
+        "libelleEdition": "du groupe Les Républicains",
+        "libelleAbrege": "LR",
+        "libelleAbrev": "LR",
+        "viMoDe": {"dateDebut": "2017-06-27", "dateAgrement": None, "dateFin": None},
+        "organeParent": None,
+        "chambre": None,
+        "regime": "5ème République",
+        "legislature": "15",
+        "secretariat": {"secretaire01": None, "secretaire02": None},
+        "positionPolitique": "Opposition",
+    },
+    "PO744107": {
+        "@xsi:type": "OrganeParlementaire_Type",
+        "uid": "PO744107",
+        "codeType": "CNPS",
+        "libelle": "Commission spéciale chargée d'examiner le projet de loi pour un État au service d'une société de confiance",  # noqa
+        "libelleEdition": "de la commission spéciale chargée d'examiner le projet de loi pour un État au service d'une société de confiance",  # noqa
+        "libelleAbrege": "Commission spéciale sur la société de confiance",
+        "libelleAbrev": "CSCONF",
+        "viMoDe": {"dateDebut": "2017-11-27", "dateAgrement": None, "dateFin": None},
+        "organeParent": None,
+        "chambre": None,
+        "regime": "5ème République",
+        "legislature": "15",
+        "secretariat": {"secretaire01": None, "secretaire02": None},
+    },
+}
+
+SAMPLE_ACTEURS = {
```

Comment:
That's a lot of data for a conftest, maybe move these 2 dicts to a dedicated fixtures file?